apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -134,7 +134,13 @@
 #define INV_FD -3
 
 #define MTD_CONFIG_PATH "/dev/config"
+#if defined(CONFIG_ARCH_BOARD_CY4390x)
+#define MTD_FTL_PATH "/dev/mtdblock4"
+#elif defined(CONFIG_ARCH_BOARD_IMXRT1020_EVK)
+#define MTD_FTL_PATH "/dev/smart0p6"
```

Comment:
why 6?

Reply:
Userfs is registered as 6 during the partitioning process.

Reply:
In specific defconfig, it is. But it does not guarantee all of defconfig has smartfs with 6 index. To make this working, you should enable automount with mountpoint.

Reply:
In this code, the MTD_FTL_PATH define is used as a test path for reading and writing directly to the MTD block dev.

Therefore, to run tc, you need the dev path before mounting.

However, registration numbers are registered in the order defined in CONFIG_FLASH_PART_TYPE in the defconfig file. Is there a way to unify this order with other defconfigs?